torch/_dynamo/optimizations/log_args.py

```diff
@@ -67,8 +66,8 @@ def run_node(self, n: torch.fx.Node):
 
 
 def conv_args_analysis(gm: torch.fx.GraphModule, example_inputs):
-    # lowering graph
-    gm = make_fx(gm)(*example_inputs)
-    # use Interpreter to logs the args of conv
-    ConvArgsAnalysis(gm).run(*example_inputs)
-    return gm
+    def conv_arg_inner(*args):
+        fx_g = make_fx(gm)(*args)
+        return ConvArgsAnalysis(fx_g).run(*args)
```

Comment:
This is the delay to runtime trick right? Do you need to do the same thing as AOTAutograd used to do and cache the result, so you aren't repeatedly running the analysis?

Reply:
Maybe, its benchmarking only, and for logging at that, so I figure its okay to leave it like this.